src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -121,7 +121,7 @@
      *  ----------------------------------------------------------------------------- */
 
     public final ForgeConfigSpec.BooleanValue                        enableColonyProtection;
-    public final ForgeConfigSpec.BooleanValue                        turnOffExplosionsInColonies;
+    public final ForgeConfigSpec.IntValue                            turnOffExplosionsInColonies;
```

Comment:
Don't use Integers for this.

As you specified it now, this should be an enum.